jetty-server/src/test/java/org/eclipse/jetty/server/MultiPartFormInputStreamTest.java

```diff
@@ -474,6 +475,77 @@ public void testPartFileNotDeleted() throws Exception
         assertThat(tptfd.exists(), is(true));  //explicitly written file did not get removed after cleanup
         tptfd.deleteOnExit(); //clean up test
     }
+    
+    @Test
+    public void testPartFileAbsolute() throws Exception
+    {
+        MultipartConfigElement config = new MultipartConfigElement(_dirname, 1024, 3072, 50);
+        MultiPartFormInputStream mpis = new MultiPartFormInputStream(new ByteArrayInputStream(createMultipartRequestString("tpfa").getBytes()),
+            _contentType,
+            config,
+            _tmpDir);
+        mpis.setDeleteOnExit(true);
```

Comment:
Should probably add a test in here to know if the file got deleted (or not).

The temp files (note, if the Part was specifically .write(String) then it's no longer a temp file) for a MultiPartFormInputStream should only exist for as long as that MultiPartFormInputStream is live. (ie: if the MPFIS is closed, then the temp files should be deleted)

Reply:
This should be part of the followup PR on cleanup of behavior.

Reply:
Parts that have been written only to temporary files are all deleted when a Request is completed. See https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java#L1423 and https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/jetty-server/src/main/java/org/eclipse/jetty/server/MultiPartFormInputStream.java#L333.

Reply:
Yeah, that seems to be more sane.  Cleanup at the end of the Request.